stubs/docker/docker/api/daemon.pyi

```diff
@@ -11,7 +12,7 @@ class DaemonApiMixin:
         until: datetime | int | None = None,
         filters: dict[str, Any] | None = None,
         decode: bool | None = None,
-    ) -> CancellableStream: ...
+    ) -> CancellableStream[Incomplete]: ...
```

Comment:
I prefer to not type this now.

The implementation is at https://github.com/docker/docker-py/blob/a3652028b1ead708bd9191efb286f909ba6c2a49/docker/api/daemon.py#L24 and explains that with `decode`, the stream will be a stream of dicts:

```python
>>> for event in client.events(decode=True)
...   print(event)
{u'from': u'image/with:tag',
 u'id': u'container-id',
 u'status': u'start',
 u'time': 1423339459}
...
```

The change in this PR will allow that to be expressed.